examples/sprite_texture.py

```diff
@@ -88,7 +88,13 @@ def __init__(self, img):
 
     img = sprite.image
     img.angle += 1
-    img.flip_x = t % 50 < 25
+    try:
+        img.flip_x = t % 50 < 25
```

Comment:
In earlier pygame versions, it's `flipX`. The functionality is still available.

Reply:
Oh, cool, I backported it.